src/types.rs

```diff
@@ -72,6 +72,15 @@ impl From<BlockTag> for BlockSpec {
     }
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+#[serde(untagged)]
+pub enum BlockNumberOrHash {
+    /// Block by number.
+    Number(U256),
+    /// Block by hash.
+    Hash(Digest),
+}
```

Comment:
I think this is incorrect. According the the spec, block tags are also allowed:

<img width="1216" alt="image" src="https://github.com/nlordell/ethrpc-rs/assets/4210206/b2825ee2-0e62-44d9-af91-5773bff4d030">

This type shouldn't be needed.

Reply:
So I can use BlockSpec here then. I didn't notice that keywords were also accepted here.

Reply:
Will be fixed in next commit.